src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -271,4 +285,38 @@ public void enableVisibilityIcon(String whoCanSeeComment) {
         this.hasVisibilityIcon = true;
         this.whoCanSeeComment = whoCanSeeComment;
     }
+
+    public String getCommentGiverName() {
+        return commentGiverName;
+    }
+
+    public void setCommentGiverName(String giverEmail) {
+        if (Const.DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT.equals(giverEmail)) {
+            this.commentGiverName = Const.DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT;
+            return;
+        }
+        this.commentGiverName = instructorEmailNameTable.get(giverEmail);
+    }
+
+    public String getCommentLastEditorName() {
+        return commentLastEditorName;
+    }
+
+    public void setCommentLastEditorName(String giverEmail) {
```

Comment:
The use of `set` for these method names may be misleading, as it suggests a behaviour of this form:

```java
public void setAttributeOfClass(String input) {
    this.attributeOfClass = input;
}
```

ie. not processing the parameter supplied to the method. Perhaps naming it as something of the form `setCommentLastEditorNameFromEmail` might clarify this better.